src/components/ClusterManager.js

```diff
@@ -270,7 +271,21 @@ export default class ClusterManager extends PureComponent {
     const appName = isRStudioImage ? 'RStudio' : 'terminal'
     const appLaunchLink = Nav.getLink('workspace-app-launch', { namespace, name, app: appName })
 
+    const app = currentApp(apps)
+
     return div({ style: styles.container }, [
+      app && h(Clickable, {
+        style: { display: 'flex', marginRight: '2rem' },
+        onClick: () => {
+          // TODO galaxy: implement this action
```

Comment:
Could we put a ticket number here? I think we want this to launch the galaxy cluster, so maybe TODO (SATURN-1855)? (And then add it as part of the AC to that ticket?)